docs/apis/spod-copy-move-api.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Microsoft 365 Copy and Move API (CreateCopyJobs)
+ms.author: jhendr
+author: JoanneHendrickson
+manager: pamgreen
+audience: ITPro
+f1.keywords:
+- NOCSH
+ms.topic: article
+ms.service: sharepoint-online
+localization_priority: Priority
+ms.collection: 
+- SPMigration
+- M365-collaboration
+search.appverid: MET150
+description: "Microsoft 365 Copy and Move API (CreateCopyJobs)"
+---
+# Microsoft 365 Copy and Move API (CreateCopyJobs)
+
+>[!Important]
+>SharePoint & OneDrive Copy and Move API replaces the CreateCopyJob API.  The CreateCopyJob API has been deprecated. 
+
+
+The following API is based on the use of the SharePoint Client Side Object Model (CSOM). We recommend using [NuGet](https://www.nuget.org/) packages when you reference CSOM in your solution. 
+
+You can find the latest version of the SharePoint Online CSOM package from the [NuGet library](https://www.nuget.org/) using the ID *Microsoft.SharePointOnline.CSOM*. 
+
+>[!Important]
+>Learn more about [limitations](#limitations) of this API before you begin.
+
+
+## Method
+
+### CreateCopyJobs
+
+This method creates a new copy or move job that lets you  copy or move a file or folder from one site in SharePoint, OneDrive, or Teams, to another site. 
+
+#### Syntax
+
+```cs
+public List<SPCopyMigrationInfo> CreateCopyJobs(Uri[] exportObjectUris, Uri destinationUri, SPCopyMigrationOptions options)
+ ```  
+
+#### Parameters
+
+|**Parameter**|**Description**|
+|:-----|:-----|
+|exportObjectUris|The URL of a file or folders in a list that you want to copy or move|
```

Comment:
Suggest making these parameters inline code style, surrounding with single ` because the are code